packages/Upgrade/src/api/controllers/ExperimentController.ts

```diff
@@ -370,6 +370,37 @@ export class ExperimentController {
     return this.experimentService.create(experiment, currentUser);
   }
 
+  /**
+   * @swagger
+   * /experiments/batch:
+   *    post:
+   *       description: Generate New Experiments
+   *       consumes:
+   *         - application/json
+   *       parameters:
+   *         - in: body
+   *           name: experiments
+   *           required: true
+   *           schema:
+   *             type: Array of Experiment object
+   *             $ref: '#/definitions/Experiment'
+   *           description: Experiment Structure
+   *       tags:
+   *         - Experiments
+   *       produces:
+   *         - application/json
+   *       responses:
+   *          '200':
+   *            description: New Experiment is created
+   */
+
+  @Post('/batch')
+  public createTestExperiments(
+    @Body({ validate: { validationError: { target: false, value: false } } }) experiment: Experiment[]
+  ): Promise<Experiment[]> {
+    return this.experimentService.createTestExperiments(experiment);
```

Comment:
Change this name also

Reply:
renaming createTestExperiments to  createMultipleExperiments